INSTALL.txt

```diff
@@ -56,6 +56,13 @@ you wish to communicate with the Diaspora network.
 	- For alternative server configurations (such as Nginx server and MariaDB
 	database engine), refer to the wiki at https://github.com/friendica/friendica/wiki
 
+This guide will walk you through the manual installation process of Friendica.
+If this is nothing for you, you might be interested in
+
+* the Friendica Docker image (https://github.com/friendica/docker) or
+* how install Friendica with YunoHost] https://github.com/YunoHost-Apps/friendica_ynh).
```

Comment:
What is the ```]``` doing there?

Reply:
Leftover from C&P operations from the Markdown file.